README.md

```diff
@@ -46,57 +34,274 @@ npm start
+
+### Installing / configuring a preset
+
+In order to install or configure a preset you need to add it as a dependency to your application:
+
+```bash
+$ yarn add hops-redux
+# OR npm install --save hops-redux
+```
+
+Sometimes presets have `peerDependencies` which need to be installed as well - take a look at the [indiviual preset sections](#available-presets) or watch out for peer dependency warnings in your terminal.
+
+By default Hops will find installed presets automatically (only when it is listed as a top-level dependency in your `package.json` - we won't attempt to find presets in dependencies of dependencies).
+
+If you prefer, you can also explicitly list the presets that you want to use under the `presets` key in your [application configuration](#configuration).
+
+```json
+{
+  "hops": {
+    "presets": ["hops-preset-defaults", "hops-react", "hops-redux"]
+  }
+}
+```
+
+Some presets require (or allow) additional configuration. Read the sections below for each of the presets you are using to find out what options are available to you.
+
+### Available presets
+
+In addition to the core presets that [we mentioned earlier](#presets) these following presets are available to help you with the struggles of universal rendering, deployments, service workers and such.
+
+#### `hops-redux`
+
+This preset will set-up a Redux store, take care of dehydration / rehydration and wrapping your application in a `<Provider />`.
+
+Install it to your project:
+
+```bash
+$ yarn add hops-redux react-redux redux redux-thunk
+```
+
+And pass your reducers as [render options](https://github.com/xing/hops/tree/next/packages/redux#render-options):
+
+```javascript
+import { render } from 'hops-react';
+export default render(<MyApp />, { redux: { reducers: myReducers } });
+```
+
+For more details and more advanced use-cases, head over to the [full readme of `hops-redux`](https://github.com/xing/hops/tree/next/packages/redux).
+
+#### `hops-graphql`
+
+This preset will create an [Apollo client](https://www.apollographql.com/docs/react/) and take care of dehydration / rehydration and wrapping your application in an `<ApolloProvider />`.
+
+Install it to your project:
+
+```bash
+$ yarn add hops-graphql graphql-tag react-apollo
+```
+
+And configure your GraphQL endpoint URI:
+
+```json
+{
+  "hops": {
+    "graphqlUri": "https://www.graphqlhub.com/graphql"
+  }
+}
+```
+
+Now you can use GraphQL in your project. Check out [this integration test](https://github.com/xing/hops/blob/next/packages/spec/integration/graphql) or [the `hops-graphql` template](https://github.com/xing/hops/tree/next/packages/template-graphql) to see usage examples and head over to [the `hops-graphql` readme](https://github.com/xing/hops/tree/next/packages/graphql) for more details.
+
+#### `hops-postcss`
+
+This preset will enable PostCSS support via a css-loader and add the PostCSS Preset Env to your project.
```

Comment:
I think it should be explicitly mentioned that this preset also enables support for `css modules`